.github/workflows/reusable-deployment.yml

```diff
@@ -166,6 +166,49 @@ jobs:
           slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
           sls-stage: ${{ steps.setup.outputs.sls-stage}}
 
+  analytics-algolia-settings:
+    needs: [crn-sls-deploy]
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment-name }}
+    container:
+      image: ghcr.io/yldio/asap-hub/node-python-sq:86a189edc900d4e1afdcf3935c697292f69d409b
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cache build typecheck output
+        uses: ./.github/actions/cache-unplugged
+        with:
+          environment-name: ${{ inputs.environment-name }}
+      - name: Setup
+        id: setup
+        uses: ./.github/actions/setup-environment
+        with:
+          environment-name: ${{ inputs.environment-name }}
+      - name: Cache transpile output
+        uses: ./.github/actions/cache-build
+        with:
+          cache-prefix: ${{ steps.setup.outputs.cache-prefix-transpile }}
+          environment-name: ${{ inputs.environment-name }}
+      - name: Get Algolia Keys
+        id: algolia-keys
+        uses: ./.github/actions/algolia-keys
+        with:
+          algolia-api-key-name: ${{ steps.setup.outputs.crn-algolia-api-key-name }}
+          algolia-app-id-name: ${{ steps.setup.outputs.crn-algolia-app-id-name }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-default-region: ${{ steps.setup.outputs.aws-default-region }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Synchronize Schema
+        run: |
+          yarn algolia:set-settings -a $ALGOLIA_APP_ID -k $ALGOLIA_API_KEY -n $ALGOLIA_INDEX -p analytics
```

Comment:
same goes for settings - you could just rename ALGOLIA_INDEX to something like ALGOLIA_ENTITY_INDEX and add ALGOLIA_ANALYTICS_INDEX or something like that and have them run step after step